dash-info.yaml

```diff
@@ -1,10 +1,10 @@
-pkg_help_description: >
-    Dash DataTable is an interactive table component designed for
-    designed for viewing, editing, and exploring large datasets.
-    DataTable is rendered with standard, semantic HTML <table/> markup,
-    which makes it accessible, responsive, and easy to style. This
-    component was written from scratch in React.js specifically for the
-    Dash community. Its API was designed to be ergonomic and its behaviour
-    is completely customizable through its properties.
-pkg_help_title: >
+pkg_help_description: >-
```

Comment:
Is this file generated?

Reply:
@Marc-Andre-Rivet The `dash-info.yaml` file is "handwritten", so it's something we'll carry forward but won't update using the package generator.

Reply:
Ok, so what I see is that this code (and other similar files) will need to be merged back into `dev` once merged into master or be committed into `dev` first, merged into `master` and this branch updated.

Reply:
Right, fortunately the only non-build artifacts are these YAML files (I believe), so I'll cherry-pick those into `dev` and remove from this PR before we merge.

Reply:
fixed via 

https://github.com/plotly/dash-table/pull/669
https://github.com/plotly/dash-html-components/pull/147
https://github.com/plotly/dash-core-components/pull/722